okhttp-tests/src/test/java/com/squareup/okhttp/CacheTest.java

```diff
@@ -113,41 +118,50 @@
     // assertCached(false, 100);
```

Comment:
Quick summary of changes:

Newly cached with this update:
204 No Content: cool
206 Partial Content: **I'd prefer to leave this as-is.** Implementing this is difficult (and incomplete).
404 Not Found: cool
405 Method Not Allowed: cool
410 Gone: cool
414 URI too long: cool
501 Not implemented: cool

Newly not cached:
308 Permanent Redirect. **I'd prefer to leave this as-is**. [RFC 7238](https://tools.ietf.org/html/rfc7238) says, _"A 308 response is cacheable by default"_